src/styles/global.css

```diff
@@ -12,6 +12,7 @@ html {
   --primary-green-dark: #206a6f;
   --primary-green-light: #a9db91;
   --primary-red-error: #b30b0b;
+  --primary-red-error-light: #ffebee;
```

Comment:
let's not add new colors. 

I think you can use the same color schema as here for the missingData.module. That way it's easier to refactor once we get the proper design
<img width="584" alt="Screenshot 2024-08-14 at 14 06 11" src="https://github.com/user-attachments/assets/65bf3bb7-b242-4d42-83de-07fddbafeb87">